galaxy_ng/locale/fr/LC_MESSAGES/django.po

```diff
@@ -396,476 +356,512 @@ msgstr "Un espace de nom nommé %s existe déjà."
 
-#: app/constants.py:203
+#: app/constants.py:223
 msgid "Push a new containers to any namespace in the system."
-msgstr ""
+msgstr "Ajouter un nouveau conteneur dans n'importe quel espace de nom du système."
 
-#: app/constants.py:207
+#: app/constants.py:227
 msgid "Manage container namespace roles"
-msgstr ""
+msgstr "Supprimer l'espace de nom du conteneur"
 
-#: app/constants.py:208
+#: app/constants.py:228
 msgid "Manage container namespace roles."
-msgstr ""
+msgstr "Supprimer l'espace de nom du conteneur."
 
-#: app/constants.py:209
+#: app/constants.py:229
 msgid "Manage container namespace roles existing in the system."
-msgstr ""
+msgstr "Gérer les rôles de l'espace de noms des conteneurs existant dans le système."
 
-#: app/constants.py:213
+#: app/constants.py:233
 msgid "Add remote registry"
-msgstr ""
+msgstr "Ajouter un registre à distance"
 
-#: app/constants.py:215
+#: app/constants.py:235
 msgid "Add remote registry to the system."
-msgstr ""
+msgstr "Ajouter un registre à distance au système."
 
-#: app/constants.py:216 app/constants.py:222
+#: app/constants.py:236 app/constants.py:242
 msgid "Container Registry Remotes"
-msgstr ""
+msgstr "Type de registre des conteneurs"
 
-#: app/constants.py:219
+#: app/constants.py:239
 msgid "Change remote registry"
-msgstr ""
+msgstr "Changer le registre distant"
 
-#: app/constants.py:220
+#: app/constants.py:240
 msgid "Edit this remote registry."
-msgstr ""
+msgstr "Modifiez ce registre à distance."
 
-#: app/constants.py:221
+#: app/constants.py:241
 msgid "Change any remote registry existing in the system."
-msgstr ""
+msgstr "Modifier tout registre distant existant dans le système."
 
-#: app/constants.py:225
+#: app/constants.py:245
 msgid "Delete remote registry"
-msgstr ""
+msgstr "Supprimer le registre distant"
 
-#: app/constants.py:226
+#: app/constants.py:246
 msgid "Delete this remote registry."
-msgstr ""
+msgstr "Supprimez ce registre distant."
 
-#: app/constants.py:227
+#: app/constants.py:247
 msgid "Delete any remote registry existing in the system."
-msgstr ""
+msgstr "Supprimez tout registre distant existant dans le système."
 
-#: app/constants.py:231
+#: app/constants.py:251
 msgid "Change task"
-msgstr ""
+msgstr "Changer la tâche"
 
-#: app/constants.py:232
+#: app/constants.py:252
 msgid "Edit this task."
-msgstr ""
+msgstr "Modifier cette tâche."
 
-#: app/constants.py:233
+#: app/constants.py:253
 msgid "Edit any task existing in the system."
-msgstr ""
+msgstr "Modifier toute tâche existante dans le système."
 
-#: app/constants.py:234 app/constants.py:240 app/constants.py:246
+#: app/constants.py:254 app/constants.py:260 app/constants.py:266
 msgid "Task Management"
-msgstr ""
+msgstr "Jobs de gestion"
 
-#: app/constants.py:237
+#: app/constants.py:257
 msgid "Delete task"
-msgstr ""
+msgstr "Supprimer la tâche"
 
-#: app/constants.py:238
+#: app/constants.py:258
 msgid "Delete this task."
-msgstr ""
+msgstr "Supprimer ce nœud."
```

Comment:
```suggestion
msgstr "Supprimer ce tâche."
```

?

Reply:
probably more:

Reply:
The perks of having a French speaker on the team! :heart: